### DISCUSSION TITLE
Quick inline page-structure creation for known hierarchies

### DISCUSSION THREAD

**Discussion by supertoothy** [zim-desktop-wiki/zim-desktop-wiki#2120] [Ideas]
I find it easy to create page hierarchies for known structures. This is useful for certain types of notes where the page-structure and hierarchy is generally known or pre-determined, such as when working on Projects, or developing content for say, a curriculum.

In Zim, I can setup a page hierarchy with one line. For example - I can type this on a page - `Project:People:Tasks`  and Zim will create a nicely nested set of parent, child and grandchild.

Project
---People
------Tasks

I wish there was some way to create siblings. Using the example above - `Project:People:Task1.Task2.Task3`

Project
---People
------Task1
------Task2
------Task3

Do you think this will be useful to others as well?

**Comment by introt** [comment]
I've been meaning a write a custom tool or a plugin that turns hierarchical bullet lists into subpages of the current page, as this is how I usually split up my reading materials/create an annotated ToC of them. Feel free to steal the idea :)
